### PR TITLE
chore(weight): add weights to add_key_to_msa

### PR DESCRIPTION
**Pull request by enddynayn**
Add weights to extrinsic add_key_to_msa.

issue-36

### CODE REVIEW

File: pallets/msa/src/benchmarking.rs [hunk 1]
```diff
@@ -84,5 +94,14 @@ benchmarks! {
 		}
 	}: _ (RawOrigin::Signed(provider), delegator_msa_id)
 
+	add_key_to_msa {
+		let s in 1 .. 1005;
```
Comment (aramikm) [inline pallets/msa/src/benchmarking.rs:98]: `s` is not used anywhere. You can remove it

**Review by aramikm — APPROVED** [review]
Looks good for initial benchmarking.

File: pallets/msa/src/weights.rs [hunk 2]
```diff
@@ -95,9 +103,9 @@ impl WeightInfo for () {
 	// Storage: Msa KeyInfoOf (r:1 w:1)
 	// Storage: Msa MsaKeysOf (r:1 w:1)
 	fn create(s: u32, ) -> Weight {
-		(18_317_000 as Weight)
-			// Standard Error: 0
-			.saturating_add((4_000 as Weight).saturating_mul(s as Weight))
+		(68_720_000 as Weight)
```
Comment (shannonwells) [inline pallets/msa/src/weights.rs:106]: 😬 that escalated quickly

**Review by shannonwells — APPROVED** [review]
nothing to add besides deleting the unused variable.

**Pull request merged**